packages/app/ui/components/MessageInput/InputMentionPopup.tsx

```diff
@@ -10,21 +9,32 @@ import MentionPopup, { MentionPopupRef } from '../MentionPopup';
 function InputMentionPopupInternal(
   {
     containerHeight,
-    showMentionPopup,
+    isMentionModeActive,
     mentionText,
     groupMembers,
     onSelectMention,
+    setHasMentionCandidates,
   }: PropsWithRef<{
     containerHeight: number;
-    showMentionPopup: boolean;
+    isMentionModeActive: boolean;
     mentionText?: string;
     groupMembers: db.ChatMember[];
     onSelectMention: (contact: db.Contact) => void;
+    setHasMentionCandidates?: (has: boolean) => void;
   }>,
   ref: MentionPopupRef
 ) {
+  useEffect(() => {
+    if (mentionText) {
+      const filteredMembers = groupMembers.filter((member) =>
```

Comment:
don't we do this logic somewhere already with fuzzy search?

Reply:
We do it in MentionPopup itself, but we don't use fuzzy, just the .filter() method with some regexes. I'll drill the `setHasMentionCandidates` prop into there to avoid the duplicate calculation.

Reply:
fixed in latest commit.